msvc/mono.def

```diff
@@ -1,6 +1,4 @@
 ; file generated by create-windef.pl
-EXPORTS
```

Comment:
This looks wrong. @lateralusX looks like it was removed from the generation script in https://github.com/mono/mono/commit/c371f4f33c612770fc1265f5b69e35df2ba29a1f#diff-3395136676c42e59a79d21aff7b16bafL33, was this intentional?